index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <meta name="description" content="Author: Fernando Silva, School: Microverse, Project: MyPortfolio-Milestone-2" />
+
+  <link rel="stylesheet" href="./styles.css">
+
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" />
+
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link rel="preload" as="style" onload="this.onload = null; this.rel = 'stylesheet'"
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" />
+
+  <link rel="preload" as="style" onload="this.onload = null; this.rel = 'stylesheet'"
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" />
+
+  <title>Portfolio</title>
```

Comment:
Suggestion: title tag is suggested to add between meta and link to make a bit clear for SEO and necessary links.